core/tests/data/third_party/js/third_party.js

```diff
@@ -138075,7 +138075,7 @@ angular.module('ui.scroll', []).directive('ngScrollViewport', [
 										if (result.length === 0) {
 											eof = true;
 											adapter.bottomPadding(0);
-											console.log('appended: requested ' + bufferSize + ' records starting from ' + next + ' recieved: eof');
+											console.log('appended: requested ' + bufferSize + ' records starting from ' + next + ' received: eof');
```

Comment:
I don't think we should be editing this file. It is autogenerated.

Reply:
Whoops my bad, I put reverted the changes for this file. Thanks Nithesh!